apps/core/serializers.py

```diff
@@ -165,3 +165,43 @@ def validate(self, attrs):
             )
 
         return attrs
+
+
+class PasswordChangeSerializer(serializers.ModelSerializer):
+    password = serializers.CharField(
+        write_only=True,
+        required=True,
+        validators=[validate_password],
+    )
+    password2 = serializers.CharField(
```

Comment:
The name password2 can be confusing as we don't have 2 passwords, we only have one password and repeat it to confirm.
Therefore, it is better to name the variable repeated_password/confirmed_password